### PR TITLE
Actions for protocols website

### PR DESCRIPTION
**Pull request by ElsLommelen**
This PR adds 3 actions to be used in inbo/protocolsource in order to automate the checking and rendering of protocols.  See [here](https://github.com/ElsLommelen/protocolsource/pull/6) for a detailed description.

### CODE REVIEW

File: protocol_check/action.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+      We recommend using a service account with the least permissions necessary.
+      Also when generating a new PAT, select the least scopes necessary.
+
+
+      [Learn more about creating and using encrypted secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)
+    default: ${{ github.token }}
+runs:
+  using: 'docker'
+  image: docker://inbobmk/protocols
```
Comment (hansvancalster) [inline protocol_check/action.yml:19]: no entrypoint needed here?

Comment (ElsLommelen) [reply protocol_check/action.yml:19]: No, I don't think so.  The dockerfile in protocolsource refers to this entrypoint (`entrypoint_check.sh`), so this is the default when running the docker.

**Review by hansvancalster — DISMISSED** [review]
Thanks for adding these actions. Looks OK to me.

File: protocol_check/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# protocol_check
+
+Dedicated Github action to check protocols in repository `github.com/inbo/protocolsource` for shortcomings by running R script `docker/check_all.R` in this repository.
```
Comment (hansvancalster) [inline protocol_check/README.md:3]:
```suggestion
Dedicated Github action to check protocols in repository `github.com/inbo/protocolsource` for shortcomings by running `protocolhelper::check_all()` in `entrypoint_check.sh`.
```

**Comment by hansvancalster** [comment]
@ElsLommelen you will need to change the base for this PR from inbo:master to inbo:main.  I don't have the rights to do so.

File: protocol_website/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# protocol_website
+
+Dedicated Github action to render new protocols in repository `github.com/inbo/protocolsource` using `protocolhelper::render_release()` and add the website to repository `github.com/inbo/protocols`.
```
Comment (hansvancalster) [inline protocol_website/README.md:3]:
```suggestion
Dedicated Github action to render new protocols in repository `github.com/inbo/protocolsource` using `protocolhelper:::render_release()` and add the website to repository `github.com/inbo/protocols`.
```

File: protocol_website/README.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+In both repositories (protocolsource and protocols), a tag is added to the last commit using (`protocolhelper::set_tags()`).
```
Comment (hansvancalster) [inline protocol_website/README.md:4]:
```suggestion
In both repositories (protocolsource and protocols), a tag is added to the last commit using (`protocolhelper:::set_tags()`).
```

File: protocol_update/README.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# protocol_update
+
+Dedicated Github action to add information on the new protocol to `.zenodo.json` and the general `NEWS.md` in repository `github.com/inbo/protocolsource` by running `protocolhelper` functions `update_zenodo()` and `update_news_release()`.
```
Comment (hansvancalster) [inline protocol_update/README.md:3]:
```suggestion
Dedicated Github action to add information on the new protocol to `.zenodo.json` and the general `NEWS.md` in repository `github.com/inbo/protocolsource` by running `protocolhelper` internal functions `update_zenodo()` and `update_news_release()`.
```

**Comment by hansvancalster** [comment]
@ThierryO this one is ready to be merged. Can you merge it (we do not have write access)? Thanks!

**Pull request merged**